### ISSUE TITLE
Diff Image do not show any diff change. 

### ISSUE THREAD

**Issue by jambudi** [pointfreeco/swift-snapshot-testing#404]
We implemented successfully snapshot tests for critical scenarios using XCUITests and snapshots. When I run locally using Fastlane or Xcode. it is working fine. Currently, all tests are failing on builds machining We are not able to see any diff change in diff image. I have set a 0.985 match. and I also tested 1 match locally and works fine as well.

<img width="426" alt="Screen Shot 2020-11-03 at 4 43 41 PM" src="https://user-images.githubusercontent.com/25522050/97978643-db64b200-1df3-11eb-8095-a0d88cbafef7.png">

- Builds machines and local simulators are same (iPhone 11 Pro)
- We also override status bars to keep time, network, and batteries to keep the same for all snapshot
-

**Comment by mackoj** [comment]
Can you share the images please ?

**Comment by mackoj** [comment]
Have you tried using https://kaleidoscope.app in order to compare your diffs ?
This `SnapshotTesting.diffTool = "ksdiff"` will simplify using `kaleidoscope.app` for the diff process.

**Comment by jambudi** [comment]
Yes, I am using SnapshotTesting.diffTool = "ksdiff" that I am calling from functions. Still failing

**Comment by jambudi** [comment]
I also check both images on https://online-image-comparison.com and seem valid. I haven't seen any diff

**Comment by jambudi** [comment]
Even I do as: .wait(for: timeout, on: .image(precision: 0.80)), still faiing

**Comment by jambudi** [comment]
@mackoj Any help ?

**Comment by jambudi** [comment]
@gonzalezreal

**Comment by mackoj** [comment]
Have you tried to change the output and not use image but something like recursive description for exemple ?

**Comment by ShurupuS** [comment]
@jambudi It's not empty actually. You can check this by downloading diff image and using adjust color feature in mac os x preview. 
<img width="426" alt="97978643-db64b200-1df3-11eb-8095-a0d88cbafef7" src="https://user-images.githubusercontent.com/3691348/101280043-25b8c480-37e0-11eb-8917-4444c87830f0.png">

**Comment by stephencelis** [comment]
@jambudi Were you able to figure out the problem or are you able to provide more info?

**Comment by jambudi** [comment]
@stephencelis  No, We were not able to fix it. I reduce the percentage match to 0.85

**Comment by jambudi** [comment]
I can provide more information. Let me know what exactly you need.  @stephencelis

**Comment by mstana** [comment]
@ShurupuS big 👍, I am also uploading color settings that worked for me:
<img width="319" alt="Screen Shot 2021-09-24 at 2 06 21 PM" src="https://user-images.githubusercontent.com/6535588/134671521-6084486f-6a35-4567-9c39-22fbeb2c782d.png">

**Comment by mstana** [comment]
@ShurupuS big 👍, I am uploading the color settings that worked for me after hitting Auto Levels button: 
<img width="319" alt="Screen Shot 2021-09-24 at 2 06 21 PM" src="https://user-images.githubusercontent.com/6535588/134671957-c7d71588-991a-41c2-bf82-4898c4bfc2ce.png">
